### DISCUSSION TITLE
what to do?

### DISCUSSION THREAD

**Discussion by Sublime3Degree** [Chia-Network/chia-blockchain#8060] [Q&A]
have won a block on space pool, so i got 0.25 xch, but it went through to my old recieve address not in my new one, how can i retrieve that old wallet? block is 706302 any ideas?

**Comment by keliew** [comment]
Use your mnemonics of the old wallet and transfer it out. And change the receive address to your new wallet.

No other way...

**Comment by BrandtH22** [comment]
@Sublime3Degree , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
